can.yml

```diff
@@ -0,0 +1,8 @@
+generator:
+  templateDirectory: /Users/alex/code/gin-in-a-can/templates/go-gin
```

Comment:
Would you mind using a relative path here please?

Reply:
Definitely. 
Would rather not keep it in the repo altogether